src/pages/index.tsx

```diff
@@ -5,7 +5,7 @@ import HomepageHero from 'components/homepage-hero'
 import Footer from 'components/footer'
 import s from './index.module.css'
 
-function Homepage(): ReactElement {
+const WaypointHomePage = (): ReactElement => {
```

Comment:
I think this might have been an accidental rename, since I think this is the main `/` homepage!

Reply:
Oops, it was! I was renaming a few others that all had the same name and I think I accidentally grouped this one in with it too. Will fix. 😊